File: 程式語言 V2答案檔/C++/404.cpp
#include <iostream>
#include <cstring>
using namespace std;

int main()
{
    char str[50];
    int i, max, count[26] = { 0 };
    cin >> str;

    for (i = 0; i < strlen(str); i++) {
        count[str[i] - 'a']++;
    }
    max = 0;
    for (i = 1; i < 26; i++) {
        max = (count[i] > count[max]) ? i : max;
    }
    cout << (char)(max + 'a') << endl;
    cout << count[max];
    return (0);
}

File: 程式語言 V2答案檔/Go/404.go
package main
import "fmt"

func main(){
	var array [26]int
	var str string
	fmt.Scanln(&str)
	max := 0
	for i := 0; i < len(str); i++{
		array[str[i]-'a'] += 1
		if array[str[i]-'a'] > max{
			max = array[str[i]-'a']
		}
	}
	for i := 0; i < 26; i++{
		if array[i] == max{
			fmt.Println(string('a'+i))
			fmt.Println(max)
		}
	}
}
